ADBench/run-all.ps1

```diff
@@ -181,9 +209,17 @@ Class Tool {
 		elseif ($objective.endswith("SPLIT")) { $out_name = "$($this.name)_split" }
 		else { $out_name = $this.name }
 		$output_file = "${dir_out}${fn}_times_${out_name}.txt"
-		if (!$script:repeat -and (Test-Path $output_file)) {
-			Write-Host "          Skipped test (already completed)"
-			return
+		if (Test-Path $output_file) {
```

Comment:
Wouldn't it be clearer to leave `!$script:repeat -and` in the outer `if`?  Nothing is run in the body unless `!$script:repeat`.  The body would become

```powershell
if ($script:repeat_failures) { ... }
else { ... }
```

Reply:
I can go ahead and do that for you if you like.

Reply:
Good change, please do!